sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/Customizations/Models/MetricsData.cs

```diff
@@ -24,12 +24,14 @@ public MetricsData(int version, Metric metric, MetricPoint metricPoint) : base(v
                 case MetricType.DoubleSum:
                     metricDataPoint = new MetricDataPoint(metric.Name, metricPoint.GetSumDouble())
                     {
+                        Namespace = metric.MeterName,
```

Comment:
this method switches on `metric.MetricType`, provides `metric.Name` to the ctor, and then sets Namespace to `metric.MeterName`.

it might be simpler to provide `metric` to the `MetricDataPoint`.

Reply:
That would move the switch to custom ctor right?. Let me know if you mean something else.

Reply:
yeah it would, but i think it would be simpler overall. 
this is off topic for your PR, feel free to merge this and we can refactor later :)

Reply:
Makes sense. I will do a follow up.